spec/lib/strip_empty_sessions_spec.rb

```diff
@@ -1,72 +1,87 @@
 # -*- encoding : utf-8 -*-
-require File.expand_path(File.dirname(__FILE__) + '/../spec_helper')
-describe StripEmptySessions do
+require 'spec_helper'
 
+describe StripEmptySessions do
   def make_response(session_data, response_headers)
     app = lambda do |env|
       env['rack.session'] = session_data
       return [200, response_headers, ['content']]
     end
-    strip_empty_sessions = StripEmptySessions
-    app = strip_empty_sessions.new(app, {:key => 'mykey', :path => '', :httponly => true})
-    response = Rack::MockRequest.new(app).get('/', 'HTTP_ACCEPT' => 'text/html')
+
+    app = StripEmptySessions.new(app, key: 'mykey', path: '', httponly: true)
+
+    Rack::MockRequest.new(app).
+      get('/', 'HTTP_ACCEPT' => 'text/html', lint: true)
```

Comment:
`Rack::MockRequest#get` only accepts a [`Symbol` key for `:lint`](https://github.com/rack/rack/blob/2.2.0/lib/rack/mock.rb#L20) and it would be gross to make `'HTTP_ACCEPT'` a Symbol, so ignoring this.